### ISSUE TITLE
what parameters need to be modified with the current train.py on WorldExpo'10, ucsd

### ISSUE THREAD

**Issue by lorren5136** [leeyeehoo/CSRNet-pytorch#77]
I would like to ask what parameters need to be modified with the current train.py training data set WorldExpo'10, ucsd, such as: learning rate, transform and some other parameters. Is there something recurring? I never get the results in the thesis after training.
